new datastruct workshop (#217)
NuevoFoundation/workshops: content/english/datastruct/Arraylist/example.md

public class Main {
  public static void main(String[] args) {
    ArrayList<String> order = new ArrayList<String>();
    order.add("Whopper");
    order.add("Chicken Nugget");
    order.add("Fries");
    order.remove(2);
    //removes fries from the arraylist
  }
}
```
Similar to `set`, the `remove` method removes an element at an index number.

<hr>

<h1>Size of ArrayList</h1>

```js javascript
import java.util.ArrayList;

public class Main {
  public static void main(String[] args) {
    ArrayList<String> order = new ArrayList<String>();
    order.add("Whopper");
    order.add("Chicken Nugget");
    order.add("Fries");
    System.out.println(order.size());
    //outputs the size of the arraylist, in this case: 3
  }
}
```
`size()` returns the size of the arraylist.

<hr>

<h1>Iterating through an Arraylist</h1>

```js javascript
import java.util.ArrayList;

public class Main {
  public static void main(String[] args) {
    ArrayList<String> order = new ArrayList<String>();
    order.add("Whopper");
    order.add("Chicken Nugget");
    order.add("Fries");
    for (int i = 0; i < order.size(); i++) {
      System.out.println(order.get(i));
    }
  }
}
```

> By using a combination or the for loop and the `get` method, we can interate through the arraylist and print out each element at every index.

Reviewer: ```suggestion
By using a combination of the `for` loop and the `get()` method, we can iterate through the arraylist and print out each element at every index.
```
